core-services/image-mirroring/knative/mapping_knative_knative-nightly_serving_quay

```diff
@@ -0,0 +1,26 @@
+registry.ci.openshift.org/openshift/knative-serving-activator quay.io/openshift-knative/knative-serving-activator
```

Comment:
We'd ideally keep the old files to have both images mirrored. Then, only introduce the new mirroring to new versions.

Reply:
We should rethink those according to
https://redhat-internal.slack.com/archives/CD87JDUB0/p1680071453233949?thread_ts=1680070041.097729&cid=CD87JDUB0